app/main/views/frameworks.py

```diff
@@ -1121,24 +1124,21 @@ def view_contract_variation(framework_slug, variation_slug):
         )
 
         # Send email confirming accepted
-        mandrill_client = DMMandrillClient()
-        email_body = render_template(
-            'emails/{}_variation_{}_agreed.html'.format(framework_slug, variation_slug)
+        notify_client = DMNotifyClient(
+            current_app.config['DM_NOTIFY_API_KEY'],
+            templates=dict()
```

Comment:
`DMNotifyClient` looks for a dict of templates in the app's config during `__init__`, just like it does for the api key. We should be keeping any template IDs we end up creating in the app config - this line would overwrite that config, and we'd never find our framework's template ID in the try/except below.

Reply:
This is a mistake on my part. I need to write the template, but your comment speaks to a wider thing - should the templates be part of the config rather than inserted on initialisation?

Reply:
Yes - that's the pattern we use across the apps.